packages/turbo-repository/rust/src/internal.rs

```diff
@@ -34,6 +36,8 @@ pub(crate) enum Error {
     },
     #[error("Package directory {0} has no parent")]
     MissingParent(AbsoluteSystemPathBuf),
+    #[error("Package graph error")]
+    PackageGraphError,
```

Comment:
This implements `From<turborepo::package_graph::Error>` for you and will forward the underlying error message.
```suggestion
    #[error("Package graph error: {0}")]
    PackageGraph(#[from] turborepo_repository::package_graph::Error),
    #[error("package.json error: {0}")]
    PackageJson(#[from] turborepo_repository::package_json::Error),
```